frontend/src/reducers/parcelsReducer.spec.tsx

```diff
@@ -52,6 +52,7 @@ const defaultParcelDetail: IParcelDetail = {
     zoning: '20',
     zoningPotential: '21',
     agencyId: '22',
+    municipality: '23',
```

Comment:
It's currently a free-form string value, not a number.

Reply:
Please note the above is the string "23", not a number so it complies. :)